apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx

```diff
@@ -35,6 +35,32 @@ export const FirebaseFeatureFlagsProvider = ({ children }: Props): JSX.Element =
   const localOverrides = useSelector(overriddenFeatureFlagsSelector);
   const dispatch = useDispatch();
 
+  const getAllFlags = () => {
+    if (remoteConfig) {
+      return getAll(remoteConfig);
+    }
+
+    return null;
+  };
+
+  const getAllCurrencyFlags = (): Record<string, Feature> | null => {
+    const allFlags = getAllFlags();
+
+    if (allFlags) {
+      const entries = Object.entries(allFlags)
+        .filter(([key]) => {
+          return key.includes("_currency_");
```

Comment:
Would it work with `feature_currency_` instead ? To me, the more specific we are, the better.

(Example that I have in mind, if we decide to do the remote config with a `config_currency_XXX` 🤷‍♂️ )